src/dirs.proj

```diff
@@ -12,6 +12,22 @@
     <Warning Text="Looks like there may be an update to the build tools. Please run build.cmd from the root of the repo to refresh the build tools." /> 
   </Target>
 
+  <ItemGroup>
+    <!-- Include test runtime when restoring packages in EnsureDependencies -->
+    <PackageConfigs Include="$(BaseIntermediateOutputPath)test-runtime\packages.config" />
+  </ItemGroup>
+  
+  <Target Name="PrepareTestRuntimeRestore" BeforeTargets="EnsureDependencies">
+    <Copy
+      SourceFiles="$(ToolsDir)test-runtime-packages.config"
+      DestinationFiles="$(BaseIntermediateOutputPath)test-runtime\packages.config"
+      SkipUnchangedFiles="$(SkipCopyUnchangedFiles)"
+      OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
+      Retries="$(CopyRetryCount)"
+      RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)" />
+  </Target>
```

Comment:
I'm not sure I understand this. So what you are really doing here is a rename? If that is the case should we just rename the file in the tools package and avoid this?